tests/schema_update_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build omni
```

Comment:
As a 2nd thought. If we use the slim tag, I guess we can avoid specifying the build tag. I feel that would actually be better @d-bytebase @tisonkun ?

Reply:
> I guess we can avoid specifying the build tag.

How to? You may create a patch or PR based on this PR to show the way.

IIUC integration tests WILL fail if there is no embedded db env, but if you don't specify the build tag, it will run despite whether there is an embedded db, and thus there IS a situation these tests fail.

Reply:
Yeah, that's what I meant. The goal is to remove the go:build tag on these tags by taking !slim or omni as the default.

Reply:
@d-bytebase I already enable "Allow edits and access to secrets by maintainers".

Reply:
Done. I also made some other refactoring changes. PTAL. Thanks!